OPA.ps1

```diff
@@ -92,18 +101,14 @@ function Get-ExeHash {
     }
 
     $Hash = ($(Get-Content $OutFile -raw) -split " ")[0]
+    Write-Debug "Downloaded expected hash of $Hash for version $ExpectedVersion of $FileName"
     Remove-Item $OutFile
 
     return $Hash
 }
 
 function Confirm-OPAHash {
     param (
-        [Parameter(Mandatory)]
-        [ValidateNotNullOrEmpty()]
-        [Alias('out')]
-        [string]$OPAExe,
```

Comment:
Did you test this with a different download file name than the default, it was previously requested that the user can indicate the name they want to download the exe file as & that was stored in the `$OPAExe` var.

Reply:
I did not.  I can but assumes the OPA re-write effort tested.  I did have to adjust a bit so I can go back and test.

Reply:
This has been fixed, resolving